core/envelope-sim/src/main/kotlin/fr/sncf/osrd/envelope_sim/etcs/ETCSBrakingSimulator.kt

```diff
@@ -36,15 +43,29 @@ interface ETCSBrakingSimulator {
 
     /** Compute the ETCS braking curves for each LoA, ordered by LoA offset. */
     fun computeSlowdownBrakingCurves(
-        mrsp: Envelope,
+        envelope: Envelope,
         limitsOfAuthority: Collection<LimitOfAuthority>
     ): LOABrakingCurves
 
     /** Compute the ETCS braking curves for each EoA, ordered by EoA offset. */
     fun computeStopBrakingCurves(
-        mrsp: Envelope,
+        envelope: Envelope,
         endsOfAuthority: Collection<EndOfAuthority>
     ): EOABrakingCurves
+
+    /**
+     * Compute the ETCS LoAs, for every slowdown on an ETCS portion of the path present in the
+     * envelope.
+     */
+    fun computeLoaLocations(mrsp: Envelope): List<LimitOfAuthority>
+
+    /**
+     * Compute the ETCS EoAs, for every stop on an ETCS portion of the path present in the envelope.
```

Comment:
To clarify that it filters:
```suggestion
     * Compute the ETCS EoAs, only for stops that are inside an ETCS portion of the path present in the envelope.
```